### ISSUE TITLE
Issue with `react-diff-viewer-continued`

### ISSUE THREAD

**Issue by bwp91** [boazpoolman/strapi-plugin-config-sync#105]
Hey

Not an issue with this plugin per se, but building strapi fails on a new install with the config sync plugin due to:

https://github.com/Aeolun/react-diff-viewer-continued/issues/31

I needed to override the version in my package json like this:

```json
  "dependencies": {
    "@strapi/admin": "4.14.4",
    "@strapi/plugin-documentation": "4.14.4",
    "@strapi/plugin-i18n": "4.14.4",
    "@strapi/plugin-users-permissions": "4.14.4",
    "@strapi/provider-upload-aws-s3": "4.14.4",
    "@strapi/strapi": "4.14.4",
    "strapi-plugin-config-sync": "^1.2.0",
  },
  "overrides": {
    "strapi-plugin-config-sync": {
      "react-diff-viewer-continued": "3.2.6"
    }
  },
```

for strapi to build successfully.

Maybe this will help someone else in the meantime until that ^ repo is fixed. Hopefully this issue can be closed soon!

**Comment by boazpoolman** [comment]
Hmm odd.

This plugin explicitly uses version 3.2.6 of the `react-diff-viewer-continued` as per the yarn.lock anyways. See:
https://github.com/boazpoolman/strapi-plugin-config-sync/blob/master/yarn.lock#L4876

So I wouldn't know why that override would be necessary

**Comment by boazpoolman** [comment]
Also just two days ago a succesfull build of Strapi (with config-sync installed) can be witnessed in Github Actions.
Using Strapi 4.14.4 and config-sync 1.2.0.

https://github.com/boazpoolman/strapi-plugin-config-sync/actions/runs/6523104347/job/17713405843

**Comment by boazpoolman** [comment]
Allright. Publishing .lock files is not really a thing.

That leaves us with 2 options:
1. Wait for `react-diff-viewer-continued` to release a patch and in the meantime use your workaround
2. Explicitly set the version constraint of `react-diff-viewer-continued` to `3.2.6` in the package.json of the plugin.

I'm gonna wait a bit to see if they'll come up with a patch any time soon.
Otherwise I'll go with option 2 in the near future.

**Comment by boazpoolman** [comment]
I've locked the version of `react-diff-viewer-continued` to `3.2.6` in PR #107.

To prevent this from going unnoticed in the future I've updated the integration tests to install the plugin dependencies with the `--no-lockfile` flag. By doing that the dependency install job will behave just as it would if somebody would install the plugin from NPM (without a lockfile). And because of that we can replicate an issue like this in the pipeline. See:
https://github.com/boazpoolman/strapi-plugin-config-sync/actions/runs/6551531539/job/17792933173
